pxf/build.gradle

```diff
@@ -122,6 +124,47 @@ subprojects { subProject ->
 }
 
 project('pxf-service') {
+
+// Copy existing sources and replace any occurrences of @tokenName@ with desired values
+    task generateSources {
+        doFirst {
+            copy {
+                from('src/main/java') {
+                    include '**/*.java'
```

Comment:
Instead include the specific class file. We will never hardcode the version in multiple class files